runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/LeftRecursion/JavaExpressions.stg

```diff
@@ -17,18 +17,18 @@ expressionList
     :   e (',' e)*
     ;
 e   :   '(' e ')'
-    |   'this' 
+    |   'this'
     |   'super'
     |   INT
     |   ID
-    |   type '.' 'class'
+    |   typespec '.' 'class'
```

Comment:
:exclamation: A new test needs to be added to ensure `type` remains a valid rule name in the Java target.